cellmlmanip/units.py

```diff
@@ -338,24 +339,52 @@ def summarise_units(self, expr: sympy.Expr):
         logger.debug('summarise_units(%s) ⟶ %s', expr, found.units)
         return found.units
 
-    def get_conversion_factor(self, quantity, to_unit):
-        """Returns the magnitude multiplier required to convert from_unit to to_unit
+    def get_conversion_factor(self,
+                              to_unit,
+                              from_unit=None,
+                              quantity=None,
+                              expression=None):
+        """Returns the magnitude multiplier required to convert a unit to the specified unit.
+
+        Note this will work on either a unit, a quantity or an expression. If more than one
```

Comment:
Hang on,
this is inconsistent. as later on you use an assert to say that at least 2 of those should be None. 
Personally I think that makes more sense than accepting arguments that you're not using, but either way the description should be consistent with what's actually happening.

Reply:
Good catch - thanks